### DISCUSSION TITLE
[mpvnet] 黑米奇修改版SVP Manager的冲突问题

### DISCUSSION THREAD

**Discussion by TokuSeiR** [hooke007/mpv_PlayKit#82] [libmpv]
是这样的，我这边在mpv下，大佬你的懒人包自带的svpflow是可以正常补帧的，但介于补帧效果和所需性能的问题我这边想用svp manager来进行补帧，但我无论是使用懒人包内置svpflow还是官方的svpflow都无法让svp manager生效，当然mpv.conf里面的mpvpipe是打开了的，但是哪怕卡bug使svp manager的osd信息在mpv内显示，让svp manager的控制面板显示正在补帧，补帧本身也没有生效，包括简易包和svp manager内置的mpv也不会生效。所以我这边想求教一下大佬，到底是svp有问题还是VapourSynth有问题，而且会不会和基于VapourSynth的官方svp code无法压制视频也有关系，我这边还用了黑米奇的svp脚本，我昨天已经搞了一天了也没有解决，网上也查找不到任何相关教程，大佬你这边是我唯一的希望了。
这边是SVP Manager的日志[SVP.log](https://github.com/hooke007/MPV_lazy/files/7836565/SVP.log)
[SVP2.txt](https://github.com/hooke007/MPV_lazy/files/7836585/SVP2.txt)
其他的log我这边不知道怎么提取，还请大佬多包涵

**Comment by hooke007** [comment]
- 你的svp manager就没安装完全（这不一定重要）
```
10:13:34.929 [W]: Main: PYTHONPATH doesn't contain mpv64 folder, VLC and libmpv players may not work
10:13:34.930 [W]: Main: PYTHONPATH is EMPTY
```
- 你用的是mpvnet不是懒人包（这也不一定重要）
```
10:13:37.015 [I]: VideoPlayer: mpv [mpvnet.exe] connected, waiting for the video info...
```
- ....如果用manager不要再往mpvnet里塞svpflow，一定会报错冲突
```
10:13:38.229 [E]: Playback [5f76d805]: VS - Python exception: Plugin C:\Program Files (x86)\SVP 4\plugins64\svpflow1_vs.dll already loaded (com.svp-team.flow1) from C:/Program Files/mpvnet/vapoursynth64/plugins/svpflow1_vs.dll
```

**额外的如果用黑米奇的修改版，那直接移除懒人包/简易包里的vapoursynth64文件夹移除**

近似案例
https://bbs.acgrip.com/forum.php?mod=redirect&goto=findpost&ptid=5843&pid=79824&fromuid=7914


======================


> manager内置的mpv也不会生效

这是什么意思。。内置mpv无法工作那只能说明svp manager要么自己有bug要么没装全
先别整奇奇怪怪的脚本，保证内置mpv能用manager提供的预设跑起来

**Comment by TokuSeiR** [comment]
阿哲，解决了，明明我都知道大佬是VCB的，却忘了上新论坛找，麻了呀

**Comment by TokuSeiR** [comment]
感谢大佬

**Comment by TokuSeiR** [comment]
再麻烦一下大佬，svp新内置的RIFE AI怎么调效果比较好啊，或者说有没有详细的设置教程麻烦大佬指个路
![屏幕截图 2022-01-10 122356](https://user-images.githubusercontent.com/45761551/148718280-42385979-d672-4142-b097-fec4b29b1254.png)

**Reply by hooke007** [reply]
rife的效果没有调节的空间

**Reply by TokuSeiR** [reply]
那那个gpu threads和math precision是按照什么来设置的呢

**Reply by hooke007** [reply]
数字越低性能要求越低

**Reply by TokuSeiR** [reply]
哦哦明白了，谢谢大佬
